expyfun/_experiment_controller.py

```diff
@@ -940,6 +944,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             If timestamp==False, returns a list of strings indicating which
             keys were pressed. Otherwise, returns a list of tuples
             (str, float) of keys and their timestamps.
+            If releases is True, the last value in the tuple will be a string,
+            either 'press' or 'release'.
```

Comment:
we have a built-in `assert_version` function in expyfun that is tied to GitHub commit hashes, and a `download_version` method that puts the desired version of the codebase in the `./` directory relative to the script. So breaking old code is less of a danger if we assume that users are making use of these tools.  So I think we don't mind.  It's more a question about code complexity within the package; more standardized return signatures is better in my view because it means fewer conditionals elsewhere.